src/Microsoft.AspNet.Mvc/MvcOptionsSetup.cs

```diff
@@ -62,11 +63,13 @@ public static void ConfigureMvc(MvcOptions options)
 
             // Set up metadata providers
             options.ModelMetadataDetailsProviders.Add(new DefaultBindingMetadataProvider());
+            options.ModelMetadataDetailsProviders.Add(new DefaultValidationMetadataProvider());
             options.ModelMetadataDetailsProviders.Add(new DataAnnotationsMetadataDetailsProvider());
```

Comment:
`DataAnnotationsMetadataDetailsProvider` was originally the pattern here, but I think I like your suggestion better. Will change it.